
25th of septh 2015:

	added logs to createvm.py also got unexpected logs as bellow with INFO
		DEBUG:root:2015-09-20 09:57:19.344825 createVm : missed syntax  ...
		DEBUG:root:2015-09-20 10:02:06.167304 create vm called...
		INFO:urllib3.connectionpool:Starting new HTTP connection (1): controller
		DEBUG:urllib3.connectionpool:Setting read timeout to None
		DEBUG:urllib3.connectionpool:"POST /v2.0/tokens HTTP/1.1" 200 1447
		INFO:urllib3.connectionpool:Starting new HTTP connection (1): controller
		DEBUG:urllib3.connectionpool:Setting read timeout to None


